features/FEATURE_COMMON_PAL/mbed-coap/.settings/language.settings.xml

```diff
@@ -0,0 +1,195 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
That file is from the master repository, where it is intended to be present. Removing it would mean performing a transformation during import, which I assume is not otherwise happening?

I understood that imported code could generally be left as-is, relying on .mbedignore to filter out unwanted stuff from the build as necessary. Can that be the case here?